build.sbt

```diff
@@ -1209,10 +1209,12 @@ lazy val `maven-plugin` = (project in file("dev") / "maven-plugin")
       "-Dorg.slf4j.simpleLogger.showLogName=false",
       "-Dorg.slf4j.simpleLogger.showThreadName=false"
     ),
-    pomExtra :=
-      <prerequisites>
+    pomExtra ~= (existingPomExtra => {
```

Comment:
Sorry, @marcospereira ! 😞 
I did not think that a `pomExtra` already have an implicit value.